Sources/AsyncHTTPClient/HTTPHandler.swift

```diff
@@ -342,8 +342,8 @@ extension HTTPClient {
     }
 
     /// HTTP authentication
-    public struct Authorization {
-        private enum Scheme {
+    public struct Authorization: Hashable {
+        private enum Scheme: Hashable {
```

Comment:
Why was this added?

Reply:
Getting equatable conformance on the internal Proxy Type enum.